### PR TITLE
ADR_0007 - repository release notes publishing

### PR DESCRIPTION
**Pull request by toban**

### CODE REVIEW

File: docs/adr/0007-wikibase-release-notes-publish.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+proposed
+
+## Context
+
+We have in [ADR0005] adopted the process for maintaining our release notes within the repository we also want to make a decision as to where these should be published for a broader audience.
+
+As our process for maintaining release notes will be closely modeled after mediawiki it also makes sense to review how mediawiki is publishing theirs.
+
+The [template] used for describing the different release notes topics is based on wikitext that after a release branch is cut and the template has been populated with the relevant changes, is to serve as the base for a richer curated version on mediawiki.org [(Example: 1.35)](https://www.mediawiki.org/wiki/Release_notes/1.35).
```
Comment (mariushoch) [inline docs/adr/0007-wikibase-release-notes-publish.md:15]: "that after a release branch is cut" I think this could be clarified a little.

**Comment by mariushoch** [comment]
Looks sensible to me.

File: docs/adr/0007-wikibase-release-notes-publish.md (hunk 1 above)
Comment (toban) [reply docs/adr/0007-wikibase-release-notes-publish.md]: I updated this section a bit but still not going into too much detail on how the branches are cut, is this sufficient?

File: docs/adr/0007-wikibase-release-notes-publish.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+proposed
+
+## Context
+
+We have in [ADR0005] proposed a process for maintaining our release notes within the repository we also want to make a decision as to where these should be published for a broader audience.
+
+As our process for maintaining release notes will be closely modeled after mediawiki it also makes sense to review how mediawiki is publishing theirs.
+
+As described in [ADR0005], the mediawiki/core team is maintaining release notes for each specific release within the corresponding release branch. During the preparation of creating a new release from the development branch, a new release branch is cut and the [template] that has been populated with the relevant changes from the ongoing development is to serve as the base for a richer curated version on mediawiki.org [(Example: 1.35)](https://www.mediawiki.org/wiki/Release_notes/1.35).
```
Comment (mariushoch) [inline docs/adr/0007-wikibase-release-notes-publish.md:15]: > the mediawiki/core team

There's no mediawiki core team… it's kind of anyone's responsibility to keep them in shape.

**Review by tarrow — APPROVED** [review]

**Pull request merged**